site/src/pages/docs/advanced-spring-boot-integration.mdx

```diff
@@ -0,0 +1,421 @@
+# Using Spring Boot with Armeria
+
+Spring framework provides powerful features which are necessary for building a web application, such as
```

Comment:
```suggestion
Spring framework provides powerful features that are necessary for building a web application, such as
```

I heard that what comes after 'which' is something that's OK to omit completely. https://www.grammarly.com/blog/which-vs-that/